src/components/ComponentDetails/PropTable/PropDescription/index.tsx

```diff
@@ -1,12 +1,17 @@
 import React from "react";
 import CodeAttribute from "../../../CodeAttribute";
+import "./index.css";
 
-const PropDescription: React.FC<{ description: string; type: string }> = ({
-  description,
-  type,
-}) => (
-  <div>
+const PropDescription: React.FC<{
+  description: string;
+  type: string;
+  required: boolean;
+}> = ({ description, type, required }) => (
+  <div className="prop-description">
     <ic-typography>{description}</ic-typography>
+    {required === true && (
```

Comment:
You shouldnt need the `=== true` as the type is a boolean 
This should do the job already `{required && (`